webapp/src/main/java/io/github/microcks/util/grpc/GrpcTestRunner.java

```diff
@@ -220,6 +269,17 @@ public void checkServerTrusted(X509Certificate[] certs, String authType) {
       return results;
    }
 
+   private static Metadata convertHeadersToMetadata(Set<Header> headers) {
+      Metadata metadata = new Metadata();
+      for (Header header : headers) {
+         log.info("Adding header {} to request", header.getName());
```

Comment:
Maybe `INFO` level is a bit high here as these are details. Could you lower this to `DEBUG`?

Reply:
Absolutely. Additionally, it was an unnecessary logging message only meant for debugging purposes for me during development. I removed it completely